### ISSUE TITLE
Unable to see URLs in multiple sitemaps

### ISSUE THREAD

**Issue by kishori-p** [nuxt-modules/sitemap#227]
Hello Team,

I am trying to get the sitemaps created using multiple sitemap configuration.
As a result, multiple sitemaps are getting created, but I am not able to see the URLs in it.

Here are the sitemap are getting created:
<img width="500" alt="Screenshot 2024-01-31 at 8 46 53 AM" src="https://github.com/nuxt-modules/sitemap/assets/158249155/ceb6cde1-cd5e-4532-95a5-8f82d2d1687d">

When I try to open a particular sitemap file, I am getting this:

<img width="1297" alt="Screenshot 2024-01-31 at 8 47 04 AM" src="https://github.com/nuxt-modules/sitemap/assets/158249155/b43145fb-2e72-4a37-8c85-4ac8c59ff41a">

This is my Nuxt.config.ts

```
modules: ["@nuxtjs/i18n", "@nuxtjs/sitemap",],
i18n: {
    defaultLocale: "en",
    locales: ["en", "fr", "ko", "ja", "zh_hans"],
    strategy: "prefix",
},
sitemap: {
    sitemaps: true,
    autoI18n: false,
    defaultSitemapsChunkSize: 5
}
```

**Comment by kishori-p** [comment]
Hi team, I am not sure, if I am doing anything wrong here or any configuration is missing.

**Comment by harlan-zw** [comment]
I've tried to replicate this without this issue, can you provide a reproduction on StackBlitz?

You can use this as a starter: https://stackblitz.com/edit/nuxt-starter-jwuie4?file=app.vue

**Comment by kishori-p** [comment]
@harlan-zw 
Here's the StackBlitz example: https://stackblitz.com/edit/nuxt-starter-qt9un8?file=nuxt.config.ts
When you can see the sitemaps created starting from 0-sitemap.xml, 1-sitemap.xml and so on.
My issue is, when I try to open 0-sitemap.xml file, it gives me the error which I have shared a screenshot above.
How can I see which URLs are included in 0-sitemap.xml?

**Comment by harlan-zw** [comment]
They seem to show fine in StackBlitz? Note, that you can't click into the link on Stackblitz, you will need to manually paste the path `/0-sitemap.xml`

**Comment by kishori-p** [comment]
@harlan-zw 
I tried copy paste the path in different tab, and I got to know that instead of `https://`, the URLs are forming with `http://`.
I am running the app on `https://` with ssl cert.
So making it `https://` manually in different tab worked for me.
Do you have any idea that why the sitemap included urls with `http://`?
Also, I want to understand the use of `site` property in Nuxt.config.ts.

**Comment by harlan-zw** [comment]
Thanks, looks like a bug with Nuxt Site Config, have pushed up a fix with https://github.com/harlan-zw/nuxt-site-config/commit/cc2bfe0.

Will be available in v5.1.1
